desktop/main.js

```diff
@@ -179,7 +179,6 @@ const mainWindow = (() => {
             }
 
             if (ELECTRON_ENVIRONMENT.isDev()) {
-                require('dotenv').config();
```

Comment:
`dotenv` was added to the `desktop/start` script so that we always have it loaded in `development`